htlcswitch/mailbox.go

```diff
@@ -108,8 +109,13 @@ type memoryMailBox struct {
 	htlcPkts *list.List
 	pktIndex map[CircuitKey]*list.Element
 	pktHead  *list.Element
-	pktMtx   sync.Mutex
-	pktCond  *sync.Cond
+
+	addPkts  *list.List
```

Comment:
May want to consider changing the name of the existing htlc pkt attributes which are now only for fails/settles. something like `updPkts`, `updatePkts`, or `modifyPkts` could work.

Reply:
done, use `repPkts` to indicate that they are replies